docs/DeveloperGuide.md

```diff
@@ -138,43 +139,90 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
+The expected hours to complete is 5 and the schedule need to be between 01-01-2020 0000 and 23-12-2020 2359.
 
-1. The `execute` method of `LogicManager` is called when a user keys in an input into the application and `execute` takes in the input.
-2. The `parseCommand` method of `ProductiveNusParser` parses the user input and returns an initialized `ScheduleCommandParser` object and further calls the `parse` method of this object to identify keywords and prefixes in the user input.
-3. If user input is valid, it returns a `ScheduleCommand` object, which takes in a predicate. (`ExpectedHours` in this example user input)
-4. There is return call to `LogicManager` which then calls the overridden `execute` method of `ScheduleCommand`.
-6. The `execute` method returns a `ScheduleResult` object.
+1. `execute ("schedule 3 hrs/5 af/01-01-2020 0000 by/23-12-2020 2359")` of `LogicManager` calls the `parseCommand` method of `ProductiveNusParser`.
+2. `parseCommand("schedule 3 hrs/5 af/01-01-2020 0000 by/23-12-2020 2359")` parses the String `"schedule 3 hrs/5 af/01-01-2020 0000 by/23-12-2020 2359"` and returns an initialized `ScheduleCommandParser` object.
+3. `parseCommand("schedule 3 hrs/5 af/01-01-2020 0000 by/23-12-2020 2359")` calls the `parse` method in `ScheduleCommandParser` which parses the user input into `targetIndex`, `expectedHours`, `doAfter` and `doBefore`.
+ This is done by calling the methods `parseIndex`, `parseTime` and `parseExpectedHours` of `ParserUtil`.
+4. If the inputs are valid, a `ScheduleCommand` object will be returned, which takes in 1 for `targetIndex`, 2 for `expectedHours`, 01-01-2020 0000 for `doAfter` and 12-12-2020 0000 for `doBefore`
+5. There is a return call to `LogicManager` which then calls the overridden `execute` method of `ScheduleCommand`.
+6. The assignment in the displayed assignment list corresponding to `targetIndex`, `assignmentToSchedule` will be retrieved by calling the `get` method.
+7. The `List<Task>` containing all tasks, `taskList` will be retrieved by calling `getFilterTaskList`, then the assignment will be removed from the task list because it is not needed.
+8. A new schedule will be created by calling `createValidSchedule` that takes in the `assignmentToSchedule` and `taskList`.
+9. A new assignment `scheduledAssignment` will be created by calling `createScheduledAssignment` with `assignmentToSchedule` and `schedule`.
+10. `setAssignment` method of `model` will be called to replace `assignmentToSchedule` with `scheduledAssignment`.
+11. The `execute` method returns a `CommandResult` object with `scheduledAssignment`. 
+
+Given below is the sequence diagram for the interactions within `LogicManager` for the `execute ("schedule 3 hrs/5 af/01-01-2020 0000 by/23-12-2020 2359")` API call.
+
+   ![Sequence Diagram for ScheduleCommand](images/ScheduleSequenceDiagram.png)
+   <br/>*Figure X: Sequence Diagram for ScheduleCommand*
```

Comment:
Hi Minh! Can you help to refer to all the other figures in the DG and then renumber your diagrams accordingly?